07-advanced-git/README.md

```diff
@@ -1 +1,6 @@
 # WIP
+
+Sections:  
+
+7. Advanced Git
+   1. `Git Log`
```

Comment:
Placed in so that people know that the `Git Log` chapter is here

Reply:
Added a comment below